package-res/resources/web/pentaho/data/filter/AbstractFilter.js

```diff
@@ -86,10 +76,7 @@ define([
     },
 
     /**
-     * Returns a filter that is the intersection of this filter with another.
-     * In other words, implements the INTERSECT operation between this filter and another.
-     * @param {} other -
-     * @returns {*}
+     * @inheritdoc
```

Comment:
Running ant-js does not show any docs, because you moved the description to an interface.